packages/next/src/server/typescript/rules/config.ts

```diff
@@ -101,7 +101,7 @@ const API_DOCS: Record<
     },
     link: 'https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#revalidate',
     isValid: (value: string) => {
-      return value === 'false' || Number(value) >= 0
+      return value === 'false' || Number(value.replaceAll(/(\d)(_)(\d)/g, '$1$3')) >= 0
```

Comment:
```suggestion
      return value === 'false' || Number(value.replace(/_/g, '')) >= 0
```

Let's simplify this a bit.

Reply:
@ijjk This can cause problems, for example `_123` will be recognized as the static value `123` whereas it references the variable `_123`.